### ISSUE TITLE
Create code tutorial and model weights for eyes open vs eyes closed

### ISSUE THREAD

**Issue by CYHSM** [DeepMReye/DeepMReye#13]
Current notebooks only support regression targets, therefore we should:
- Add Jupyter notebook with code for model training eyes open vs. eyes closed (or other classification targets)
- Add model weights to OSF

**Comment by alexayue** [comment]
how to compute the proportion of time spent eyes closed for each volume of each participant.

**Comment by CYHSM** [comment]
We extracted blinks (and blink duration) from ground truth eye-tracking data and then calculated the proportion of time spent eyes-closed as a percentage of the duration of one TR. For example, let's say we have a TR of 1s and the participant spent their eyes closed 0.2s during this TR (which is based on the number of blinks within this TR), then we have a proportion of 1/5 or 20%.

**Comment by alexayue** [comment]
I want to classify the eye open and eye closed using MRI images, how to achieve this by deepmreye?

**Comment by Naubody** [comment]
Thanks for your question and interest, @alexayue.

1) The eyes-open vs. closed classification model is slightly different than the model for gaze position (e.g. different loss function...). The repository does currently not include a notebook for the eyes-open vs. closed model, but we plan to add one and will notify you here when it is online.

2) We have much less training data for the eyes-open vs. closed classification and therefore do not have a pre-trained version of the model that is expected to work well across datasets and scanning sequences. This means you would likely need your own training data. Do you have those?

**Comment by Naubody** [comment]
@alexayue, do you need any further help with this or can the issue be closed?
